Sources/OpenAPIServiceModel/CreateOpenAPIServiceModel.swift

```diff
@@ -0,0 +1,382 @@
+                               modelOverride: ModelOverride?,
+                               document: OpenAPI.Document) {
+        let (members, bodyStructureName) = getOperationMembersAndBodyStructureName(
+            operation: operation,
+                            operationName: operationName,
+                            model: &model,
+                            modelOverride: modelOverride, document: document)
+        
+        setOperationInput(bodyStructureName: bodyStructureName, operationInputMembers: members,
+                          description: &description, model: &model, operationName: operationName)
+        
+        setOperationOutput(operation: operation, operationName: operationName, model: &model,
+                           modelOverride: modelOverride, description: &description, document: document)
+    }
+    
+    static func getOperationMembersAndBodyStructureName(
+            operation: OpenAPI.Operation,
+            operationName: String,
+            model: inout OpenAPIServiceModel,
+            modelOverride: ModelOverride?, document: OpenAPI.Document) -> (members: OperationInputMembers, bodyStructureName: String?) {
+        var members = OperationInputMembers()
+        var bodyStructureName: String?
+        
+        if let requestBody = operation.requestBody?.b {
+            getBodyOperationMembers(requestBody, bodyStructureName: &bodyStructureName,
+                                    operationName: operationName, model: &model, modelOverride: modelOverride, document: document)
+        } else {
+            fatalError("no request body")
+        }
+        
+        for (index, parameter) in operation.parameters.enumerated() {
+            switch parameter {
+            case .b(let value):
+                switch value.context.location {
+                case .cookie, .header, .path, .query:
+                    if let schemaValue = value.schemaOrContent.schemaValue {
+                        getFixedFieldsOperationMembers(fixedFields: value, operationName: operationName,
+                                                       index: index, members: &members, items: schemaValue,
+                                                       model: &model, modelOverride: modelOverride)
+                    }
+                }
+            case .a(_):
+                fatalError("Not implemented.")
+            }
+        }
+        
+        return (members: members, bodyStructureName: bodyStructureName)
+    }
+    
+    static func getBodyOperationMembers(_ request: OpenAPI.Request, bodyStructureName: inout String?,
+                                        operationName: String, model: inout OpenAPIServiceModel,
+                                        modelOverride: ModelOverride?, document: OpenAPI.Document) {
+        for (_,value) in request.content {
+            if let schema = value.schema?.b {
+                switch schema {
+                case .object:
```

Comment:
Why is object the only case here?  Does OpenAPIKit not fill out values for the other cases (like boolean, number, integer, string, etc)?

Reply:
I was just following the precedent set by SwaggerServiceModel because it seems like the swagger files for the service code only specify objects or references under the requestBody (formerly known as body or form data)

Reply:
Got it, can we comment this part of the code to make this clear?  I think someone would be a bit confused if they tried to use those values and the service model broke/didn't generate the code.

Reply:
Yes I will comment this.